docs/rules/getter-return.md

```diff
@@ -0,0 +1,80 @@
+# Enforces that a return statement is present in property getters (getter-return)
+
+The get syntax binds an object property to a function that will be called when that property is looked up. It is firstly introduced in ECMAScript 5:
```

Comment:
I think the last sentence could read better. Maybe something like:
It was first introduced in ECMAScript 5